src/zapv2/ajaxSpider.py

```diff
@@ -29,13 +29,13 @@ def status(self):
         """
         This component is optional and therefore the API will only work if it is installed
         """
-        return next(self.zap._request(self.zap.base + 'ajaxSpider/view/status/').itervalues())
+        return self.zap._request_value('ajaxSpider/view/status/')
```

Comment:
This file is generated code - any manual changes will be lost when its changed. We can change the generator but I was trying to avoid that